crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -38,13 +38,6 @@ contract OnChainProposer is IOnChainProposer, ReentrancyGuard {
     /// @dev This is crucial for ensuring that only subsequents blocks are committed in the contract.
     uint256 public lastCommittedBlock;
 
-    /// @notice The next block to commit.
-    /// @dev This variable holds the block number of the next block to commit.
-    /// @dev `nextBlockToCommit` should be equal to `lastCommittedBlock` + 1.
-    /// @dev Only the block with the block number equal to `nextBlockToCommit` will be committed.
-    /// @dev This variable is called by the `l1_committer.rs`.
-    uint256 public nextBlockToCommit;
```

Comment:
Why we stopped caring about this?

Reply:
We can simply use `lastCommittedBlock + 1` and avoid duplicate data. I think that was a workaround to identify the "just initialized" state. Since we decided to not commit nor verify the genesis, it's no more needed